backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/scope/model/impl/ElasticsearchScopeSearchIndexesContext.java

```diff
@@ -101,6 +102,36 @@ public ElasticsearchSearchFieldContext field(String absoluteFieldPath) {
 		return resultOrNull;
 	}
 
+	@Override
+	public ElasticsearchIndexSchemaNamedPredicateNode namedPredicate(String absoluteNamedPredicatePath) {
+		ElasticsearchIndexSchemaNamedPredicateNode resultOrNull;
+		if ( elements().size() == 1 ) {
+			resultOrNull = indexModels.iterator().next().filterNode( absoluteNamedPredicatePath );
+		}
+		else {
+			resultOrNull = createMultiIndexFilterNode( absoluteNamedPredicatePath );
+		}
+		if ( resultOrNull == null ) {
+			throw log.unknownNamedPredicateForSearch( absoluteNamedPredicatePath, indexesEventContext() );
+		}
+		return resultOrNull;
+	}
+
+	@Override
+	public boolean hasSchemaObjectNodeComponent(String absoluteFieldPath) {
+		for ( ElasticsearchIndexModel indexModel : indexModels ) {
+			AbstractElasticsearchIndexSchemaFieldNode field = indexModel.fieldOrNull( absoluteFieldPath );
+			// Even if we have an inconsistency with the Lucene backend,
+			// we decide to be very lenient here,
+			// allowing ALL the model incompatibilities Elasticsearch allows.
+			if ( field != null && field.isObjectField() ) {
+				return true;
+			}
+		}
+
+		return false;
+	}
```

Comment:
It looks like a git conflict resolution went wrong. This has been removed in master, please remove it here.